app/assets/javascripts/tasks_index.coffee

```diff
@@ -88,21 +89,43 @@ intializeAdvancedFilter =->
       $advancedFilterActive.val(false)
       $drop.removeClass('fa-caret-up').addClass('fa-caret-down')
     else
-      $search.css("border-bottom-left-radius","0px")
+      $search.css("border-bottom-left-radius", "0px")
       $advanced.css("border-bottom-right-radius", "0px")
       $advancedFilterActive.val(true)
       $drop.removeClass('fa-caret-down').addClass('fa-caret-up')
 
   if $('#order_param')
-    #    $('#' + order.value).addClass('active')
+#    $('#' + order.value).addClass('active')
     $('#order_created').addClass('active')
 
-    #  $('#order_rating').click ->
-    #    $('#order_rating').addClass('active')
-    #    $('#order_created').removeClass('active')
-    #    document.getElementById('order_param').value = 'order_rating'
+  #  $('#order_rating').click ->
+  #    $('#order_rating').addClass('active')
+  #    $('#order_created').removeClass('active')
+  #    document.getElementById('order_param').value = 'order_rating'
```

Comment:
Can / should we remove these line? They don't seem to work, currently (I re-added them, of course)

Reply:
I plan to add the rating next, so I would leave it there for now

Reply:
Alright, fine for me -- thanks for clarifying!